### ISSUE TITLE
Overall look-and-feel update

### ISSUE THREAD

**Issue by tonyahowe** [LiteratureInContext/LiC-app#387]
We had discussed re-doing the look and feel of the site. Here is a link to a frontpage that might make sense for us, if you think it works: [https://lic-4be695.webflow.io/](https://lic-4be695.webflow.io/)

I'm not sure how the other pages inside the site would look, but we are open to suggestions. I also hope we can keep the image you found for the branding--the floral period-specific image. Perhaps as a small band at the very top and very bottom? Something to maintain the branding.

**Comment by wsalesky** [comment]
@tonyahowe Do we want to change the navigation options to match this new design as well?

**Comment by tonyahowe** [comment]
Yes, I think so. I didn’t put anything in the template, yet. Let me know if you need more details!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by tonyahowe** [comment]
Oh, rats, the copy paste didn’t come through—sorry about that. I’ll get it to you tomorrow.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by tonyahowe** [comment]
Psych--I did it tonight. I had to recreate the discussion notes, because I didn't hit "comment" on the issue I was writing.... I think this mostly captures it: 

Nav Structure

The collection
- overview of contents/mission [with embedded collection graph]
- browse and search [remove browse/search by publication place]
- custom coursepacks

Data Visualizations
- Timeline
- Map
- People and places
- Collection graph

For teachers
- Custom coursepacks
- Assignment samples
- Project templates
- Documentation

About
- publications
- Contributors
- Join us

Log in button

——- 
Front page

Search box like hathitrust.org
[ search full collection by keyword.  Options/Search ]
X advanced search

- Search takes to page browse and search like current results page
- Advanced search takes to more granular search like hathitrust.org
   - fields: in all,  in author, in title, in annotations, in headnotes
   - Limits by: date, has a head note, has facsimile page images, has audio

Front page content: Quality literary texts, all in one place 

Button for students || button for teachers
- For students: page with section about project created with students, by students, for students [link to contributors], how to use, link to coursepacks
- For teachers: page with section about the project, features, classroom sourced nature [link to become a contributor], create your own custom coursepacks,

Rest of front page much like current mockup content, but simplified and for a more general audience. Some portions can go into other pages for teachers.

**Comment by wsalesky** [comment]
@tonyahowe @jobrien62 @cruotolo First pass done. Please open new issues for enhancements or bugs.
